src/renderer/store/modules/utils.js

```diff
@@ -439,11 +439,18 @@ const actions = {
           case 'playlists':
             subPath = 'playlists'
             break
+          case 'podcasts':
+            subPath = 'podcasts'
+            break
+          case 'releases':
+            subPath = 'releases'
+            break
           case 'channels':
           case 'about':
             subPath = 'about'
             break
           case 'community':
+          case 'comments':
```

Comment:
This is not mentioned in PR description ._.

Reply:
Removed